src/Runner.Worker/Container/ContainerHooks/HookContainer.cs

```diff
@@ -0,0 +1,62 @@
+using System.Collections.Generic;
+using System.Linq;
+using Newtonsoft.Json;
+
+namespace GitHub.Runner.Worker.Container.ContainerHooks
+{
+    public class HookContainer : HookArgs
+    {
+        public string Id { get; set; }
+        public string DisplayName { get; set; }
+        public string Network { get; set; }
+        public string NetworkAlias { get; set; }
+        public string Image { get; set; }
+        public string Name { get; set; }
```

Comment:
Do we need Name?

Reply:
Is that helpful to pass to the hooks? I don't think its in the ADR. Should we just remove the field? Ideally I think we should only add fields we are planning to use.